src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Picture {
+    private final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public double calculateArea() {
+        return 1.;
+    }
```

Comment:
The `calculateArea` method should not have an implementation in the abstract class since the area calculation is specific to each figure type. This method should be abstract to force the subclasses to provide their own implementations.